proseco/guide.py

```diff
@@ -64,17 +65,18 @@ class GuideTable(ACACatalogTable):
 
     # Elements of meta that should not be directly serialized to YAML
     # (either too big or requires special handling).
-    yaml_exclude = ('stars', 'cand_guides', 'dark')
-    pickle_exclude = ('stars', 'dark')
+    yaml_exclude = ('stars', 'cand_guides', 'dark', 'known_fids')
+    pickle_exclude = ('stars', 'dark', 'known_fids')
 
     # Name of table.  Use to define default file names where applicable.
     # (e.g. `obs19387/guide.yaml`).
     name = 'guide'
 
     # Required attributes
-    required_attrs = ('att', 't_ccd_guide', 'date', 'dither_guide')
+    required_attrs = ('att', 't_ccd_guide', 'date', 'dither_guide', 'n_guide')
```

Comment:
Adding n_guide to required_attrs snuck in here, but I think it can just stay here and go with this PR unless there are objections.